<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogWindow</class>
 <widget class="QWidget" name="DialogWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>190</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>190</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>190</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:qlineargradient(spread:pad, x1:0.025, y1:0.385909, x2:0.99, y2:0.374364, stop:0 rgba(255, 18, 3, 255), stop:1 rgba(76, 16, 255, 255))
}</string>
  </property>
  <widget class="QLabel" name="Name_Label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans MT Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color: rgb(255, 255, 255);
background-color:rgb(255, 170, 255)rgb(85, 0, 127)
}</string>
   </property>
   <property name="text">
    <string>Name the preset</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="Name_Input">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>391</width>
     <height>28</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans MT Condensed</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
background-color: rgb(203, 203, 203);
border-radius: 10px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans MT Condensed</family>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background-color:rgb(203, 203, 203);
border:none;
color:rgb(0, 0, 0);
padding-top:5px;
text-align: center;
border-radius:15px
}
QPushButton:hover{
background-color:rgb(158, 158, 158);
}
QPushButton:pressed{
background-color:rgb(158, 158, 158);
}</string>
   </property>
   <property name="text">
    <string>Save preset</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
